src/css/_main-layout.scss

```diff
@@ -93,83 +93,80 @@
 // shadows
 .settings,
 .toolbar {
-  box-shadow: 0 0px 11px rgba(0, 0, 0, 0.4);
+  box-shadow: 0 0 11px rgba(0, 0, 0, .4);
 
   @media (min-width: 640px) {
-    box-shadow: 0 4px 11px rgba(0, 0, 0, 0.3);
+    box-shadow: 0 4px 11px rgba(0, 0, 0, .3);
   }
 }
 
 // initial state
-  .toolbar {
-    transform: translateY(-110%);
+.toolbar {
+  transform: translateY(-110%);
 
-    &.transition {
-      transition: transform 0.2s ease-out;
-    }
+  &.transition {
+    transition: transform .2s ease-out;
+  }
 
-    &.active {
-      // translateZ required to make Safari layer in the right order
```

Comment:
I removed these comments because it seems this is needed for at least Firefox too